### DISCUSSION TITLE
Externally Administered Multi-Sigs

### DISCUSSION THREAD

**Discussion by gregfromstl** [nation3/court#13] [Specs]
Opening this thread as a discussion spot for multi-sig implementations that satisfy the requirements found on [DeWork](https://app.dework.xyz/nation3/develop-1?taskId=3cafc59c-1e10-46d3-b6c8-466b1e05779a)

**Comment by gregfromstl** [comment]
@xPi2 Not seeing any multi-sigs with a "non-signer owner". It'd be really easy to extend an existing one to include something like it, but the tricky part is "Allow the owner to override transactions / suspend operations". We'd probably want some sort of delay in the execution of transactions so the owner has a chance to veto before it becomes executable by the signers. An alternative could be to only allow the owner to execute transactions, but in our use case would require the DAO to vote "not to veto" every transaction -- not ideal.

**Comment by xPi2** [comment]
I've been talking lately about the new Aragon Core (The replacement for the current AragonOS) with Aragon's team. It looks very promising for the kind of flexibility we need. Sadly the new version won't be ready on mainnet probably until Q4 2022 - Q1 2023.

By now, our best shot seems to be to build over [Zodiac modular architecture](https://github.com/gnosis/zodiac). I will be drafting today some doc on what we would need to do.
